Document/0x06j-Testing-Resiliency-Against-Reverse-Engineering.md

```diff
@@ -28,6 +28,17 @@ None of these measures can assure a 100% effectiveness, as the reverse engineer
 
 For example, preventing debugging is virtually impossible. If the app is publicly available, it can be run on an untrusted device that is under full control of the attacker. A very determined attacker will eventually manage to bypass all the app's anti-debugging controls by patching the app binary or by dynamically modifying the app's behavior at runtime with tools such as Frida.
 
+The techniques discussed below will allow you to detect various ways in which an attacker is attacking your application. Since these techniques are documented publicly, they are generally easy to bypass. Using an open-source detection tehcniques is a good first step in imrpoving the resiliency of your application, but they are typically easily bypassed by standard anti-detection tools. Commercial products do typically offer a higher resilience, as they will combine multiple techniques, such as:
```

Comment:
```suggestion
The techniques discussed below will allow you to detect various ways in which an attacker may target your app. Since these techniques are publicly documented, they are generally easy to bypass. Using open-source detection techniques is a good first step in improving the resiliency of your app, but standard anti-detection tools can easily bypass them. Commercial products typically offer higher resilience, as they will combine multiple techniques, such as:
```